model/validation.go

```diff
@@ -467,22 +468,16 @@ func ValidateSubnet(subnet string) error {
 // validateCIDR checks that a string is in "CIDR notation"
 func validateCIDR(cidr string) error {
 	// We expect a string in "CIDR notation", i.e. a.b.c.d/xx form
-	parts := strings.Split(cidr, "/")
-	if len(parts) != 2 {
-		return fmt.Errorf("%q is not valid CIDR notation", cidr)
-	}
-
-	var errs error
-
-	if err := ValidateCIDRBlock(parts[1]); err != nil {
-		errs = multierror.Append(errs, err)
+	ip, _, err := net.ParseCIDR(cidr)
+	if err != nil {
+		return fmt.Errorf("/%v is not a valid CIDR block", cidr)
```

Comment:
Why the `/` as the first character in the error messages here and on line 477?

Reply:
mistype